src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -873,7 +866,7 @@ public void getCustomFields() {
             if (useBranch) {
                 key = issue.getScaDetails() == null
                         ? String.format(JIRA_ISSUE_KEY, issuePrefix, issue.getVulnerability(), issue.getFilename(), request.getBranch(), issuePostfix)
-                        : String.format(JIRA_ISSUE_KEY_3, issuePrefix, issue.getScaDetails().get(0).getFinding().getSeverity(),
+                        : String.format(SCATicketingConstants.SCA_JIRA_ISSUE_KEY, issuePrefix, issue.getScaDetails().get(0).getFinding().getSeverity(),
```

Comment:
then someone created the scaDetails -> this is a scan with sca with no findings

Reply:
please rewrite this: issue.getScaDetails().get(0) 
use strems findany findfirst or wat ever.

Reply:
I checked this issue - In case of empty case (no SCA results), we won't enter to the for loop:
for (ScanResults.XIssue issue : issues) {}

In that case no issue will be modified and no error or any exceptions in the code.
@ofersk  - I extracted out the SCA format logic to a new method